### DISCUSSION TITLE
Real-ESRGAN V0.2.3.0 Release Note

### DISCUSSION THREAD

**Discussion by xinntao** [xinntao/Real-ESRGAN#182] [Announcements]
Long long time no see😺

We have added small models for **anime videos**: 🎉

You can find demos and usages in https://github.com/xinntao/Real-ESRGAN/blob/master/docs/anime_video_model.md

:white_check_mark: We add small models that are optimized for anime videos :-)

Now we have added two models.

| Models                                                                                                                             | Scale | Description                    |
| ---------------------------------------------------------------------------------------------------------------------------------- | :---- | :----------------------------- |
| [RealESRGANv2-animevideo-xsx2](https://github.com/xinntao/Real-ESRGAN/releases/download/v0.2.3.0/RealESRGANv2-animevideo-xsx2.pth) | X2    | Anime video model with XS size |
| [RealESRGANv2-animevideo-xsx4](https://github.com/xinntao/Real-ESRGAN/releases/download/v0.2.3.0/RealESRGANv2-animevideo-xsx4.pth) | X4    | Anime video model with XS size |

This release also contains the following files:

- RealESRGANv2-animevideo-xsx2.pth
- RealESRGANv2-animevideo-xsx4.pth
- realesrgan-ncnn-vulkan-20211212-windows.zip
- realesrgan-ncnn-vulkan-20211212-macos.zip
- realesrgan-ncnn-vulkan-20211212-ubuntu.zip

<hr /><em>This discussion was created from the release <a href='https://github.com/xinntao/Real-ESRGAN/releases/tag/v0.2.3.0'>Real-ESRGAN V0.2.3.0 Release Note</a>.</em>

**Comment by Troceleng** [comment]
I have to say, I'm VERY impressed! Thank you very much for the continued work. 

How come the models are so tiny? Typical community-created 4x models are about 64 MB.

**Reply by xinntao** [reply]
Thanks ;-)

**Comment by AaronFeng753** [comment]
Some Video benchmark results:

Using [Waifu2x-Extension-GUI](https://github.com/AaronFeng753/Waifu2x-Extension-GUI/releases/latest) v3.86.01, 2X upscale, same video, same machine.

|Model|Time|Speed|
|---|---|---|
|Waifu2x-ncnn-vulkan| 31s | 13.45x |
|realesrgan-x4plus-anime| 417s | 1x |
|RealESRGANv2-animevideo-xsX2| 39s | 10.69x |
|RealESRGANv2-animevideo-xsX4| 44s | 9.48x |

**Reply by xinntao** [reply]
Thank you very much @AaronFeng753 
I will add the benchmark to the readme file.
If possible, could you also please add some visual comparisons? :-) Thanks again.

**Reply by AaronFeng753** [reply]
> Thank you very much @AaronFeng753 I will add the benchmark to the readme file. If possible, could you also please add some visual comparisons? :-) Thanks again.

I post several kinds of visual comparisons in the comments, glad to help!

**Comment by AaronFeng753** [comment]
Visual comparison: @xinntao

Using [Waifu2x-Extension-GUI](https://github.com/AaronFeng753/Waifu2x-Extension-GUI/releases/latest) v3.86.02, 2X upscale

![LongComparePic](https://user-images.githubusercontent.com/52553688/146507947-f28ab800-aae3-4cbc-8a39-bda9f24f955a.png)

Original:

![Original](https://user-images.githubusercontent.com/52553688/146508127-346c81d6-3d13-4847-b19d-1ec2466a5caa.jpg)

Waifu2x-NCNN-Vulkan:

![Waifu2x](https://user-images.githubusercontent.com/52553688/146508131-f7ae8d21-9786-43dd-b286-900b9d0dd64d.png)

realesrgan-x4plus-anime:

![realesrgan-x4plus-anime](https://user-images.githubusercontent.com/52553688/146508240-ececde15-bed0-474f-bd79-8567e7c80305.png)

RealESRGANv2-animevideo-xsX2:

![RealESRGANv2-animevideo-xsX2](https://user-images.githubusercontent.com/52553688/146508269-a1d5a1b6-17ac-4a84-846a-f266ea1d3a3c.png)

RealESRGANv2-animevideo-xsX4:

![RealESRGANv2-animevideo-xsX4](https://user-images.githubusercontent.com/52553688/146508301-18d9859c-554c-4650-881d-e2c298509830.png)

**Comment by AaronFeng753** [comment]
OG vs XSX4

https://imgsli.com/ODY1NDk

![sc](https://user-images.githubusercontent.com/52553688/146509819-d40c9f60-7ad3-4140-9d82-f6053bf633bc.jpg)

**Comment by AaronFeng753** [comment]
Waifu2x VS xsX4

https://imgsli.com/ODY1NTI

realesrgan-x4plus-anime VS RealESRGANv2-animevideo-xsX4

https://imgsli.com/ODY1NTM

xsX4 VS xsX2

https://imgsli.com/ODY1NTQ

**Comment by AaronFeng753** [comment]
GIF visual comparison: @xinntao

Using [Waifu2x-Extension-GUI](https://github.com/AaronFeng753/Waifu2x-Extension-GUI/releases/latest) v3.86.02, 2X upscale

[Original:](https://user-images.githubusercontent.com/52553688/146520340-9e2d12d2-5f64-4811-9e04-515e329f1392.gif)

![original](https://user-images.githubusercontent.com/52553688/146520340-9e2d12d2-5f64-4811-9e04-515e329f1392.gif)

[RealESRGANv2-animevideo-xsX2:](https://user-images.githubusercontent.com/52553688/146520407-dd63a975-c231-4050-b744-6d0c82ec05bc.gif)

![RealESRGANv2-animevideo-xsX2](https://user-images.githubusercontent.com/52553688/146520407-dd63a975-c231-4050-b744-6d0c82ec05bc.gif)

[RealESRGANv2-animevideo-xsX4](https://user-images.githubusercontent.com/52553688/146520554-cc995091-b6dd-4d88-ae74-333dc7077039.gif)

![RealESRGANv2-animevideo-xsX4](https://user-images.githubusercontent.com/52553688/146520554-cc995091-b6dd-4d88-ae74-333dc7077039.gif)

[realesrgan-x4plus-anime](https://user-images.githubusercontent.com/52553688/146520660-4f7e7ff0-974b-4a8f-a3ea-c7bb68b34020.gif)

![realesrgan-x4plus-anime](https://user-images.githubusercontent.com/52553688/146520660-4f7e7ff0-974b-4a8f-a3ea-c7bb68b34020.gif)

[Waifu2x](https://user-images.githubusercontent.com/52553688/146520708-26267b86-dacb-4fbf-83c8-6cf5e08a2ed7.gif)

![Waifu2x](https://user-images.githubusercontent.com/52553688/146520708-26267b86-dacb-4fbf-83c8-6cf5e08a2ed7.gif)
